src/TConsole.cpp

```diff
@@ -1575,17 +1574,17 @@ int TConsole::select(const QString& text, int numOfMatch)
     P_end.setY(mUserCursor.y());
 
     if (mudlet::debugMode) {
-        TDebug(QColor(Qt::darkRed), QColor(Qt::black)) << "P_begin(" << P_begin.x() << "/" << P_begin.y() << "), P_end(" << P_end.x() << "/" << P_end.y()
+        TDebug(Qt::darkRed, Qt::black) << "P_begin(" << P_begin.x() << "/" << P_begin.y() << "), P_end(" << P_end.x() << "/" << P_end.y()
                                                        << ") selectedText = " << buffer.line(mUserCursor.y()).mid(P_begin.x(), P_end.x() - P_begin.x()) << "\n"
-                >> 0;
+                >> mpHost;
     }
     return begin;
 }
 
 bool TConsole::selectSection(int from, int to)
 {
     if (mudlet::debugMode) {
-        TDebug(QColor(Qt::darkMagenta), QColor(Qt::black)) << "\nselectSection(" << from << "," << to << "): line under current user cursor: " << buffer.line(mUserCursor.y()) << "\n" >> 0;
+        TDebug(Qt::darkMagenta, Qt::black) << "selectSection(" << from << "," << to << "): line under current user cursor: \n\"" << buffer.line(mUserCursor.y()) << "\"\n" >> mpHost;
```

Comment:
How about single quotes instead? 

```suggestion
        TDebug(Qt::darkMagenta, Qt::black) << "selectSection(" << from << "," << to << "): line under current user cursor: \n'" << buffer.line(mUserCursor.y()) << "'\n" >> mpHost;
```

Reply:
Actually I had gone through and revised a whole load of messages - then I realised I was yielding to *creature feep* and undid them all - apart from this and one other that I didn't spot! I will in fact put that part back and leave all of that to some future PR - and that *is* needed IMVHO...!

Reply:
Even better decision!